src/utils/formatBalance.ts

```diff
@@ -5,8 +5,8 @@ export const getBalanceNumber = (balance: BigNumber, decimals = 18) => {
   return displayBalance.toNumber()
 }
 
-export const getFullDisplayBalance = (balance: BigNumber, decimals = 18) => {
-  return balance.dividedBy(new BigNumber(10).pow(decimals)).toFixed()
+export const getFullDisplayBalance = (balance: BigNumber, decimals = 18, decimalsToAppear = null) => {
```

Comment:
I think it is better in this case to leave `decimalsToAppear` as undefined.

Reply:
Yea do you mean reassign the 'null' default to 'undefined', or just un-assign the `null` default? It looks like without some default it then becomes a required function argument.

Reply:
```
(balance: BigNumber, decimals = 18, decimalsToAppear?: number)
```

Reply:
Ah yes, no idea what syntax I was playing with that it didn't like... but it wasn't this. I'll amend

Reply:
Done 👌